projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -11,6 +11,32 @@
 use Automattic\Jetpack\Status;
 use Automattic\Jetpack\Status\Host;
 
+/**
+ * Add the Action Bar display setting on the General settings page.
+ * This setting allows users to hide the Action Bar on the front end of their site.
+ * The setting is stored in the `wpcom_show_action_bar` option.
+ * The setting is displayed only if the has the wp-admin interface selected.
+ */
+function wpcomsh_wpcom_show_action_bar_settings_field() {
+	add_settings_field( 'wpcom_show_action_bar', '', 'wpcom_show_action_bar_display', 'general', 'default' );
```

Comment:
I realized that atomic sites [don't show the Action Bar](https://github.com/Automattic/wp-calypso/issues/36795) and I'm wondering whether we should check about the type before adding this setting ( at least for now ). 🤔

Reply:
Yeah, we should check if the site is Simple before running this.

Alternatively, if there are no plans to bring the action bar to Atomic sites, you could implement this logic directly in the `wpcom` repo, which only runs on Simple sites.

Reply:
There is an issue asking for the action bar in atomic sites: https://github.com/Automattic/wp-calypso/issues/36795

In order to do that, I think we should move the actionbar mu-plugin from `wpcom` repo to `jetpack-wpcom-mu-plugin` or something like that.

Reply:
>In order to do that, I think we should move the actionbar mu-plugin from wpcom repo to jetpack-wpcom-mu-plugin or something like that.

Yes. @cbravobernal is something you're working on? My linked issue with yours are kind of co-dependent.

Reply:
Not yet, I've been checking how to do it while I found this task and decided to wait. Feel free to take it if you feel more comfortable to tackle all together.

Reply:
@cbravobernal sounds good. I'll see what's decided about atomic and handle these two issues accordingly.

Reply:
> if there are no plans to bring the action bar to Atomic sites, you could implement this logic directly in the wpcom repo, which only runs on Simple sites.

I would also recommend keeping things together in the dotcom codebase for now. It will make it easier to move things around when/if we eventually bring the action bar to WoA sites.